docs/src/How To Work With Us.doc.js

```diff
@@ -0,0 +1,120 @@
+    />
+  </MainSection>,
+);
+
+card(
+  <MainSection name="Resources, Slack, and Meetings - oh my!">
+    <MainSection.Subsection
+      description={`
+**Dashboards, JIRA, and OKRs**
+If you’re curious what we’re working on, you can check out our Gestalt [Sprint Dashboard](https://jira.pinadmin.com/secure/Dashboard.jspa?selectPageId=29640), our [full backlog](https://jira.pinadmin.com/secure/RapidBoard.jspa?rapidView=1936&projectKey=PDS&view=planning.nodetail&issueLimit=100), and our [Bugs Dashboard](https://jira.pinadmin.com/secure/Dashboard.jspa?selectPageId=29639). You can also reference our [go/GestaltTracker](https://docs.google.com/spreadsheets/d/10-s5BfpbesIpRTUhJSGH4tAriI-gdXHsT-eKDCtaMQk/edit#gid=1725470630) to see the immediate, larger component changes that are in the pipeline.
+
+To see the bigger picture, you can view our [OKRs](https://coda.io/d/Pinterest-2021-EPD-OKR-Tracker_de-g0jv4ClO/Gestalt-Design_suAbV#_luyId) to understand our roadmap and priorities for each quarter. These priorities are determined through our [partnership meetings](https://paper.dropbox.com/doc/How-to-Work-with-Us--BGVO2I8uo7TY0t~zzbc5PAY7Ag-KSFgsi7Me5kEqhC7sR2VA#:h2=Meetings-and-Events), which we use to learn about component needs and coordinate with designers to determine the timelines.
+
+**Slack Channels**
+Before reaching out, take a look at our [documentation](http://gestalt.netlifyapp.com) to see if it answers your question, because it will likely get you the fastest answer. Still need help? Try searching Slack for your question, and then feel free to ask if your question hasn’t been answered in the past. You can also reference our [Communication Guidelines](https://paper.dropbox.com/doc/2CXysUNfbx9mq0b6ge53Y) for more info.
+
+[#gestalt-design](https://pinterest.slack.com/archives/C0HUV5J93) is the best way to reach the Gestalt design team for design focused questions.
+
+[#gestalt-web](https://pinterest.slack.com/archives/C13KLG5P0) is for more engineering-focused questions.
+
+**Meetings and Events**
+Our meetings, events, and timelines can be found on our [Gestalt Event Calendar](https://calendar.google.com/calendar/u/0?cid=Y19ubnVsdjdjNGRsY3RxbG1jcHVlOWVyaHFuc0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).
+
+We offer two different meetings for the community to come with questions, comments, ideas, and feedback.
+
+- Business Component Crit: Geared toward design work for our business products. [Sign up sheet and notes](https://paper.dropbox.com/doc/Meeting-notes-Ads-Design-Component-Crit--A0OnUFBJPWygCqBU2aMrhAaGAQ-9bkQ0frWwmRxkYRRq5fOm)
+- Design System Office Hours: Open space for discussing designs and getting feedback on component usage within the Pinner product. [Sign up sheet](https://paper.dropbox.com/doc/Design-systems-office-hours-signup-sheet--BGF2OO1W31I96Pss5qFZ1Pz1Ag-oBkTKg5JUNO0POAzPxNl6)`}
```

Comment:
I would recommend making each of these a Subsection with title="Dashboards, JIRA, and OKRs" (the capitalization will break but I'll fix that in another PR), title="Slack Channels", and title="Meetings and Events"

This adds them to the side nav and make them link-able

Reply:
Nice - good call. Done.